### ISSUE TITLE
Game freezes when entering replay name

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#392]
```
What steps will reproduce the problem?
1. Go to official server and play with others.
2. When game finished, save the replay, entering replay name
3. Sometimes when entering replay name which is alphabet+numbers the game 
freezes, for example input 'haha4', when entering '4' the game randomly freezes

What is the expected output? What do you see instead?

The game freezes

What version of the product are you using? On what operating system?

Hedgewars 0.9.17, Windows XP SP3 32-bit Simplified Chinese

Please provide any additional information below.

```

-----
Original issue reported on code.google.com by `acme....@gmail.com` on 24 May 2012 at 3:44

**Comment by GoogleCodeExporter** [comment]
```
BTW, when checking the task manager, I see hedgewars.exe uses all of the CPU
```

-----
Original comment by `acme....@gmail.com` on 24 May 2012 at 3:46

**Comment by GoogleCodeExporter** [comment]
```
So, not sure why this is happening, but as a workaround for you personally, you 
might want to go to settings and just enable save timestamping.
That will automatically save each game to a unique name, and you can always 
rename them later.


```

-----
Original comment by `kyberneticist@gmail.com` on 24 May 2012 at 5:33

**Comment by GoogleCodeExporter** [comment]
```
Thanks, it looks like an acceptable workaround

BTW, if it's possible I can debug the game (but recently I had uninstalled Qt 
SDK)
```

-----
Original comment by `acme....@gmail.com` on 25 May 2012 at 2:58

**Comment by GoogleCodeExporter** [comment]
```
Debugging would be helpful, sure, since no one else has reported this.
Maybe a side effect of some virus on your system or something :-p

```

-----
Original comment by `kyberneticist@gmail.com` on 25 May 2012 at 4:17

**Comment by GoogleCodeExporter** [comment]
```
I actually believe it is freezing which occurs when you leave room. It just 
could happen while you're typing replay name.
```

-----
Original comment by `unC0Rr` on 23 Aug 2012 at 1:33

**Comment by GoogleCodeExporter** [comment]
```
Ah. *that* freezing I'm certainly familiar with :)

```

-----
Original comment by `kyberneticist@gmail.com` on 24 Aug 2012 at 12:35

**Comment by GoogleCodeExporter** [comment]
```
Well, I guess if it that particular freeze, then:

1) it is fixed in 0.9.18
2) as room isn't closed when admin leaves it, it is harder to run into this 
issue anyway
```

-----
Original comment by `unC0Rr` on 27 Oct 2012 at 3:47
* Changed state: **WontFix**
